Applications/MmrMobile/Contracts/IMmrMobileClient.cs

```diff
@@ -55,6 +55,23 @@ public interface IMmrMobileClient
         Task<MmrNodeData> GetTimeSeriesFromMachine(string machineNumber, string node, DateTime from, DateTime to, int take = 1000, int skip = 0);
         #endregion
 
+        #region eventdata
+
+        /// <summary>
+        /// get all  events from the requested timespan
+        /// to discuss: we should show also events, that overlap the requested duration and adjust the timestamps to fit into this period.
+        /// 
+        /// </summary>
+        /// <param name="machineNumber"></param>
```

Comment:
The param lines should get removed when there is nothing to explain.

Reply:
I personally agree, but then we have to filter out the compiler-warnings
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591?f1url=%3FappId%3Droslyn%26k%3Dk(CS1591)
![image](https://github.com/HomagGroup/HOMAG-Connect/assets/139100771/35363968-7548-4690-99a3-d0a26f9c36b8)

Reply:
There will be no error when you remove all params.